aiida_pseudo/cli/family.py

```diff
@@ -20,21 +20,28 @@ def cmd_family():
 @cmd_family.command('show')
 @arguments.PSEUDO_POTENTIAL_FAMILY()
 @options.STRINGENCY()
+@options.UNIT(default=None)
 @options_core.RAW()
 @decorators.with_dbenv()
-def cmd_family_show(family, stringency, raw):
+def cmd_family_show(family, stringency, unit, raw):
     """Show details of pseudo potential family."""
     from tabulate import tabulate
 
     if isinstance(family, RecommendedCutoffMixin):
 
-        if stringency is not None and stringency not in family.get_cutoff_stringencies():
-            raise click.BadParameter(f'`{stringency}` is not defined for family `{family}`.', param_hint='stringency')
+        if stringency is not None:
+            try:
+                family.validate_stringency(stringency)
+            except ValueError as exception:
+                raise click.BadParameter(f'{exception}', param_hint='stringency')
 
-        headers = ['Element', 'Pseudo', 'MD5', 'Wavefunction (eV)', 'Charge density (eV)']
+        if unit is None:
+            unit = family.get_cutoffs_unit(stringency)
```

Comment:
If we go with my other suggestion to validate the default in `validate_stringency`, we can turn this into
```suggestion
        unit = unit or family.get_cutoffs_unit(stringency)
```